postprocess/filters.yml

```diff
@@ -0,0 +1,7 @@
+filters:
+  - path: ingress-nginx/10_ingress_nginx_helmchart/ingress-nginx/templates
+    type: builtin
+    filter: helm_namespace
```

Comment:
The `helm_namespace` filter is not necessary anymore with ArgoCD, as the Argo application ensures that all manifests without a `metadata.namespace` field are created in the namespace indicated by the Application object.

Instead, you'll want to create the namespace object in `component/main.jsonnet`.

Reply:
Removed